samples/Actor/ActorClient/Program.cs

```diff
@@ -67,6 +73,16 @@ public static async Task Main(string[] args)
                     Console.WriteLine($"Got Incorrect Exception from actor method invocation. Exception {ex.InnerException}");
                 }
             }
+
+            Console.WriteLine("Registering the timer and reminder");
+            await proxy.RegisterTimer();
+            await proxy.RegisterReminder();
+            Console.WriteLine("Waiting so the timer and reminder can be triggered");
+            await Task.Delay(6000);
+
+            Console.WriteLine("Making call using actor proxy to get data after timer and reminder triggered");
+            receivedData = await proxy.GetData();
+            Console.WriteLine($"Received data is {receivedData}.");
```

Comment:
Please add calls to unregister reminder and timer as well, else it will just keep the reminders firing in the dapr runtime if user runs the sample multiple times.

Reply:
@amanbha: Added deregistration at the end of the program on [00e1a9f](https://github.com/dapr/dotnet-sdk/pull/213/commits/00e1a9f8490815e10d8e47ac90e6cbd28515a08e)